Sources/Classes/core/Protocols/MiniAppMessageDelegate.swift

```diff
@@ -89,3 +96,7 @@ public class MAHostEnvironmentInfo: Codable {
         )
     }
 }
+
+public class DownloadHeaders: Codable {
+    let token: String?
+}
```

Comment:
@laubengaier Sorry, I missed seeing this before. The headers needs to be open ended and allow any number of generic String pair of headers, not just a single param.

Reply:
ok gotcha, replaced it with [String: String]